mbw.spec

```diff
@@ -20,17 +20,17 @@ make
 %install
 mkdir -p %{buildroot}/usr/bin
 mkdir -p %{buildroot}%{_mandir}/man1
-install -m 755 -o root -g root mbw %{buildroot}/usr/bin/mbw
-install -m 644 -o root -g root mbw.1 %{buildroot}%{_mandir}/man1/mbw.1
+install mbw %{buildroot}/usr/bin/mbw
+install mbw.1 %{buildroot}%{_mandir}/man1/mbw.1
 
 %clean
 rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root)
-/usr/bin/mbw
+%attr(755,-,-) /usr/bin/mbw
 %doc
-%{_mandir}/man1/mbw.1.gz
+%attr(644,-,-) %{_mandir}/man1/mbw.1.gz
```

Comment:
you're referring to mbw.1.gz, will the install command above (which refers to mbw.1) zip it? I suppose so, due to the code that's been here before but I hope that it really does work :)

Reply:
I just added the `%attr` stuff, so I don't know about changes in the compression. It complies to a working `rpm` file, which is installable and the manpage opens if I install the package, so I suppose it does work.